contracts/ServiceNodeRewards.sol

```diff
@@ -755,11 +797,40 @@ contract ServiceNodeRewards is Initializable, Ownable2StepUpgradeable, PausableU
     /// the owner.
     /// @param newMax The new maximum non-signer threshold
     function setBLSNonSignerThresholdMax(uint256 newMax) public onlyOwner {
-        require(newMax > 0, "The new BLS non-signer threshold must be non-zero");
+        if (newMax <= 0)
+            revert PositiveNumberRequirement();
         blsNonSignerThresholdMax = newMax;
         emit BLSNonSignerThresholdMaxUpdated(newMax);
     }
 
+    /// @notice Max Claim amount to use in the Exponential Moving Average check
+    /// before allowing the user to claim. If the claimed amount over 24 hours
+    /// exceeds this then the claim will fail
+    /// @param newMax The new maximum non-signer threshold
+    function setClaimEMAThreshold(uint256 newMax) public onlyOwner {
+        if (newMax <= 0)
+            revert PositiveNumberRequirement();
+        claimEMAThreshold = newMax;
+        emit ClaimEMAThresholdUpdated(newMax);
+    }
+
+    /// @notice Updates the Exponential Moving Average (EMA) with a new value
+    /// @dev This function calculates a time-weighted EMA using the provided new value and current time
+    /// @param ema The TimeWeightedEMA struct to update
+    /// @param newValue The new value to incorporate into the EMA.
+    /// @param currentTime The current timestamp, usually `block.timestamp`
+    /// @custom:formula EMA = EMA_old + α * (newValue - EMA_old)
+    /// where α = 1 - 0.5 ^ (timeDiff / halfLife)
```

Comment:
Working out the value computed in the code and simplifying, I get a value of $$\alpha$$ of:
```math
\alpha = \frac{\Delta T}{\Delta T + \tau}
```
where $$\Delta T$$ is `timeDiff` and `\tau` is `halfLife`.

This is a non-obvious value for $\alpha$ to me, though, and it needs substantially more exposition in the comments on where it comes from and why, so as to be able to consider the mathematical properties of the calculation.

Looking at various references I was able to find some justification for using $$\alpha = \frac{\Delta T}{\tau}$$ as an approximation of $$\alpha = e^{-\Delta T / \tau}$$ (the former is the first-order Taylor series expansion of the latter at 0), which isn't quite the same as this, but also the applicability of that depends on update time differences being small relative to $\tau$, while here it's easily conceivable that time delta could be much larger than $\tau$.